main.go

```diff
@@ -272,8 +290,9 @@ func main() {
 	for i := 1; i <= config.Webhook.WebhookCount; i++ {
 		// set up the registerer
 		basicConfig := webhookClient.BasicConfig{
-			Timeout:         config.Webhook.Timeout,
-			RegistrationURL: config.Webhook.RegistrationURL + "?webhook=" + strconv.Itoa(i),
+			Timeout: config.Webhook.Timeout,
+			// RegistrationURL: config.Webhook.RegistrationURL + "?webhook=" + strconv.Itoa(i),
```

Comment:
This is for being able to register multiple webhooks, but listen for them all on the same endpoint; caduceus differentiates endpoints by the url and adding a parameter (`?webhook=1`) at the end makes each webhook unique to caduceus.

Reply:
So we should go back to the original way the registration URL was being set.